.gitignore

```diff
@@ -31,3 +31,5 @@
 /IDEAS/*.pyc
 systemScript.py
 /IDEAS/Inputs
+IDEAS/funnel_comp/*
```

Comment:
funnel_comp

Reply:
@jelgerjansen both work, but I have implemented the change.